org.musicbrainz.Picard.json

```diff
@@ -1,54 +1,61 @@
-                    "url": "https://src.fedoraproject.org/repo/pkgs/rpms/sip/sip-4.19.8.tar.gz/sha512/9c0a9722a9a9dd23e2eabe751d622cdd31b376b7987b2c991f5ef4a8d11853c5c6e9f1c24a0d2d864d3990e2a45c2a159ca8511efd4a7e156a14937ef3c680f7/sip-4.19.8.tar.gz",
-                    "sha256": "7eaf7a2ea7d4d38a56dd6d2506574464bddf7cf284c960801679942377c297bc"
+                    "url": "https://src.fedoraproject.org/repo/pkgs/rpms/python-qt5/PyQt5_gpl-5.10.1.tar.gz/sha512/04cd0292f7a088f9c0d088d7c73ec96cfc40253f771dfdf194eb92be7635e6fcaf04863fa24215e9c34a096658549bb2feeb8d8fdca1e39ed87f0256ef0f790b/PyQt5_gpl-5.10.1.tar.gz",
+                    "sha256": "9932e971e825ece4ea08f84ad95017837fa8f3f29c6b0496985fa1093661e9ef"
                 },
                 {
                     "type": "file",
-                    "path": "sip-configure",
+                    "path": "pyqt-configure",
                     "dest-filename": "configure"
                 }
             ],
-            "cleanup": [
-                "/bin",
-                "/include"
-            ]
-        },
-        {
-            "name": "pyqt",
-            "sources": [
-                {
-                    "type": "archive",
-                    /* Sourceforge URLs fail intermitently, get the tarballs from Fedora, check the upstream hashes */
-                    "url": "https://src.fedoraproject.org/repo/pkgs/rpms/PyQt4/PyQt4_gpl_x11-4.12.1.tar.gz/sha512/0651816ed5a371c24827e182eac5b2a2afe67928d95a09f63bd8158117aacd3e835b1efa06d4fc0c147a28038613a5bc088bd736a1348de68199da7952299af1/PyQt4_gpl_x11-4.12.1.tar.gz",
-                    "sha256": "45d724461ae58b019b9be95076ca07e212eb73084aec9125bb254825c3a05383"
-                },
+            "modules": [
                 {
-                    "type": "file",
-                    "path": "pyqt-configure",
-                    "dest-filename": "configure"
+                    "name": "sip",
+                    "sources": [
+                        {
+                            "type": "archive",
+                            /* Sourceforge URLs fail intermitently, get the tarballs from Fedora, check the upstream hashes */
+                            "url": "https://src.fedoraproject.org/repo/pkgs/rpms/sip/sip-4.19.8.tar.gz/sha512/9c0a9722a9a9dd23e2eabe751d622cdd31b376b7987b2c991f5ef4a8d11853c5c6e9f1c24a0d2d864d3990e2a45c2a159ca8511efd4a7e156a14937ef3c680f7/sip-4.19.8.tar.gz",
+                            "sha256": "7eaf7a2ea7d4d38a56dd6d2506574464bddf7cf284c960801679942377c297bc"
+                        },
+                        {
+                            "type": "file",
+                            "path": "sip-configure",
+                            "dest-filename": "configure"
+                        }
+                    ],
+                    "cleanup": [
+                        "/bin",
+                        "/include"
+                    ]
```

Comment:
I'd rather you kept this as it was, ordered before PyQt.

I'm not a big fan of nested modules in flatpak manifests, (that doesn't work as soon as multiple modules require the same dependency) and it makes it hard to review whether you changed anything for the sip module.

Please put it back before PyQt5, let's keep this flat.